grafana-plugin/playwright.config.ts

```diff
@@ -14,7 +14,7 @@ const config: PlaywrightTestConfig = {
   testDir: './integration-tests',
   globalSetup: './integration-tests/globalSetup.ts',
   /* Maximum time one test can run for. */
-  timeout: 90 * 1000,
+  timeout: 60 * 1000,
```

Comment:
60s should be plenty of time for most tests, if we put this timeout too long, it'll take forever for the CI job to fail (especially with retries enabled).

There is currently one test, "we can create an oncall schedule + receive an alert", that genuinely takes longer (because it creates a ton of different objects). This test has already been marked w/ test.slow() which increases the timeout value by 3x just for this test.